clients/venice-client/src/main/java/com/linkedin/venice/fastclient/ClientConfig.java

```diff
@@ -440,7 +440,7 @@ public static class ClientConfigBuilder<K, V, T extends SpecificRecord> {
     private int longTailRetryThresholdForComputeInMicroSeconds = 10000; // 10ms.
 
     private boolean isVsonStore = false;
-    private StoreMetadataFetchMode storeMetadataFetchMode = StoreMetadataFetchMode.DA_VINCI_CLIENT_BASED_METADATA;
+    private StoreMetadataFetchMode storeMetadataFetchMode = StoreMetadataFetchMode.SERVER_BASED_METADATA;
```

Comment:
Can we get rid `StoreMetadataFetchMode` enum completely?

Reply:
Yup, thanks for the early review. I also wanted to remove the DVC dependency from FC.